Cargo.toml

```diff
@@ -32,3 +32,4 @@ default = [ "bin" ]
 bin = [ "clap", "pretty_env_logger", "pcap", "tokio/rt-multi-thread" ]
 
 [lib]
+crate-type = [ "rlib", "staticlib" ]
```

Comment:
I'm not familiar with FFI bindings in Rust; will this affect the binary build?

If so, I'd like for this to be behind an `ffi` feature-gate, so that it can be turned-on for `lib` usage exclusively.

Reply:
This will not effect building the binary. To build the static library for use in C, you'd just build with the ``--lib`` flag, and it will generate the static ``.a`` file. When another crate uses this library, it will take the rlib from it.

Reply:
Would there also be a benefit in adding `cdylib` to the list (dynamic library)?

Reply:
Sure. I've personally never had a use for it before, but maybe other people could benefit from putting this library on their path and dynamically linking to it.